src/EventStore.Core/Services/Transport/Grpc/ReadResponse.cs

```diff
@@ -0,0 +1,65 @@
+﻿using EventStore.Core.Data;
+
+namespace EventStore.Core.Services.Transport.Grpc; 
+
+public abstract class ReadResponse {
+	
+}
+
+public class ResolvedEventWrapper: ReadResponse {
```

Comment:
it might be nice if these were nested in the abstract base class, its a pattern we often do with the message classes

Reply:
Okay